Doc/library/ast.rst

```diff
@@ -267,7 +267,7 @@ Literals
 
 .. class:: Constant(value)
 
-   A constant value. The ``value`` attribute of the ``Constant`` literal contains the
+   A constant value. The *value* attribute of the ``Constant`` literal contains the
```

Comment:
Looking at this now, I'm not sure it's actually right (sorry for the trouble!). We mark up parameters with *italics*, but attributes with double backticks. These names are both parameters to the constructor and attributes on the resulting object, but the attribute use case is the most important, as it's more common to inspect an AST than to create AST nodes.

I'll ask on the docs Discord for more opinions.